### ISSUE TITLE
Update Conch man page

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7609]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#400|
|Type|defect|
|Created|2003-11-17 05:44:41Z|

<details><summary>Searchable metadata</summary>

```
trac-id__400 400
type__defect defect
reporter__itamarst itamarst
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1069047881000000 1069047881000000
changetime__1469356984752120 1469356984752120
version__ 
owner__ 
cc__itamarst cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
1. Cover new command line options.

2. Explain how connection sharing works and how to use it with CVS.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
I assume this is about Conch, but you didn't actually offer any indication of
that in the issue. I'm setting the title appropriately; revert if it's indeed
not what you're talking about.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="z3p's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|z3p commented|
|-|-|

```
#!html
<pre>
I just reworked it (r10759), and since it fixes both of your complains, I'm
considering this closed.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 308036106372c1527458ea9d01c725e204d3335e
```
#!CommitTicketReference repository="" revision="308036106372c1527458ea9d01c725e204d3335e"
Merge pull request #400 from twisted/8660-einarfd-ssh-connection

Author: einarfd
Reviewer: rodrigc
Fixes: #8660

Port twisted.conch.ssh.connection to Python 3
```
